Jenkinsfile

```diff
@@ -109,41 +135,41 @@ pipeline {
                     }
                     stages {
                         stage ('Building and caching new node_modules') {
-                            when {
-                                anyOf {
-                                    changeset "**/research-hub-web/package.json"
-                                    equals expected: true, actual: params.FORCE_REDEPLOY_WEB
-                                }
-                            }
+                            // when {
+                            //     anyOf {
+                            //         changeset "**/research-hub-web/package.json"
+                            //         equals expected: true, actual: params.FORCE_REDEPLOY_WEB
+                            //     }
+                            // }
```

Comment:
Could this be uncommented now?
EDIT: Ah I see, you've disabled archiving the dependencies.

Reply:
Ah yes, I disabled it temporarily because I got an error when I modified the script to use the correct branch for archiving: ERROR: Unable to find project for artifact copy: Centre for eResearch (CeR)/hub-stack-pipeline/dev
I think we need to do something with the pipeline config to allow archiving on branches other than sandbox.

Reply:
Ah I see :) Yeah, we can revisit this later